### DISCUSSION TITLE
I don't understand how Jumbo's photographers.txt works

### DISCUSSION THREAD

**Discussion by Jonseed** [adieyal/sd-dynamic-prompts#111] [Q&A] [answered]
It is just filled with lines of the countries surrounded by double underscores, but how does that tell it to look in the subdirectory for those txt files? Wouldn't they all have to have the by_country/ prefix in order to tell the script to look in the subdirectory?

**Comment by adieyal** [comment, marked as answer]
You're right - it's a bug. I've fixed it in the latest version. The reason it actually works is that wildcards are matched by name. So `__austria__` would match any file called austria.txt - which works in this case, but might also pull in unnecessary wildcards from other files called austria.txt

**Comment by Jonseed** [comment]
@adieyal I updated, but this only works now if I copy all the folders inside jumbo into my wildcards directory. What if I copy the entire jumbo folder, then it is missing the jumbo/ before artists.

**Comment by Jonseed** [comment]
Could I also just use `__jumbo/artists/Photography/by_country/**/*__` to grab all the photographers from every country?

Edit: just tried it and that works

**Comment by adieyal** [comment]
What does your wildcards folder look like? wildcards/jumbo/artists/.... Or
wildcards/artists/...?

In any case your **/* should work. I don't understand the issue with
photographers.txt though. Is there still an issue with it?

**Comment by Jonseed** [comment]
@adieyal it is the first, wildcards/jumbo/artists since I just copy the entire jumbo collection into wildcards. So photographers.txt would not work in that case, since I have the jumbo folder first, is that right?

But the **/* works.
